docs/source/rules.rst

```diff
@@ -62,16 +63,39 @@ contain such values e.g.
             type: "eq"
             pattern : "customvalue"
 
+Managing Rules
+--------------
+
+To deploy a rule, use CLI ``st2 rule create ${PATH_TO_RULE}`` command, for example:
+
+.. code-block:: bash
+
+    st2 rule create /usr/share/doc/st2/examples/rules/sample_rule_with_webhook.yaml
+
+If the rule with the same name already exists, the above command will return an error:
 
-To deploy a rule, use CLI:
+.. code-block:: bash
+    
+    ERROR: 409 Client Error: Conflict
+    MESSAGE: Tried to save duplicate unique keys (E11000 duplicate key error index: st2.rule_d_b.$name_1  dup key: { : "examples.webhook_file" })
+        
+Existing rule can be updated as follows:
```

Comment:
Say something like 
> "To update the rule, edit the rule definition file and run `st2 rule update`:"

Also, add something like this: 
>
Hint: It is a good practice to always edit the original rule file, so that keep your infrastructure in code. You still can get the rule definition from the system by `st2 rule get <rule name> -j `, update it, and load it back. 
>